packages/frontend/src/graphql/mutations/create-flow-template.ts

```diff
@@ -0,0 +1,10 @@
+import { gql } from '@apollo/client'
+
+export const CREATE_FLOW_TEMPLATE = gql`
```

Comment:
createFlowFromTemplate or createTemplatedFlow

Reply:
changed to `createTemplatedFlow` thks